kerl

```diff
@@ -943,16 +943,15 @@ _do_build() {
     fi
     if [ -n "$KERL_BUILD_DOCS" ]; then
         echo 'Building docs...'
-        release=$(get_otp_version "$2")
-        cmd="make docs DOC_TARGETS=$KERL_DOC_TARGETS"
-        release_cmd="make release_docs DOC_TARGETS=$KERL_DOC_TARGETS RELEASE_ROOT=$KERL_BUILD_DIR/$2/release_$1"
```

Comment:
Isn't the main issue that this line needs to have quotes?

Reply:
I tried to inline this command in the `if` statement, but it didn't work. For some reason the `make DOC_TARGETS="$KERL_DOC_TARGETS"` still split it and failed with `missing target chunks`.

Exporting does work nicely though.